code/modules/mob/living/simple_animal/aliens/drone.dm

```diff
@@ -280,6 +280,12 @@
 // A slightly easier drone, for POIs.
 // Difference is that it should not be faster than you.
 /mob/living/simple_animal/hostile/malf_drone/lesser
-	desc = "An automated combat drone with an aged apperance."
+	desc = "An automated combat drone with an aged appearance."
```

Comment:
at least remember to fix this regardless of balanace

Reply:
balanace